opennms-doc/guide-admin/src/asciidoc/text/config-files/opennms.properties.adoc

```diff
@@ -0,0 +1,920 @@
+
+[[file-opennms-properties]]
+=== File: opennms.properties
+
+This properties file allows you to set global properties for the opennms application.
+These properties will be set as system properties at the beginning of Java startup.
+Any properties set on the Java command-line with `-Dproperty=value` will be overridden by these values.
+
+[options="autowidth"]
+|===
+| Type             | Java property file with key value pairs
+| Location         | `${OPENNMS_HOME}/etc`
+| Restart required | yes
+|===
+
+==== General
+
+[options="header, autowidth"]
+|===
+| Property | `opennms.bin.dir`
```

Comment:
Also, is the 'Property' column necessary? People should know just from looking at the doc that this is a system property that is contained in opennms.properties. I think it just takes up space.